wikimedia/wikipedia-ios
# Adjust onboarding spacing

**Phabricator:** https://phabricator.wikimedia.org/T356206#9509493

### Notes
* This PR adjust the onboarding data and privacy view

### Test Steps
1. Run a fresh install of the app
2. Go through onboarding
3. Make sure the spacing between the label and the button is correct
